R/react.R

```diff
@@ -0,0 +1,103 @@
+#' Create a React element
+#'
+#' @param module JavaScript module to import the component from.
+#' @param name Name of the component.
+#' @param props Props to pass to the component.
+#' @param deps HTML dependencies to attach.
+#'
+#' @export
+reactElement <- function(module, name, props, deps = NULL) {
+  props <- asReactData(props)
+  reactContainer(data = ReactData(
+    type = "element",
+    module = module,
+    name = name,
+    props = dropDeps(props),
+    deps = list(getDeps(props), deps)
+  ))
+}
+
+#' Parse arguments as props
+#'
+#' @param ... Arguments to prepare for passing as props to a React component
+#'
+#' @export
+asProps <- function(...) {
+  args <- rlang::dots_list(..., .homonyms = "error")
+  named <- nzchar(names(args))
+  addChildren(props = args[named], children = unname(args[!named]))
+}
+
+#' React output
+#'
```

Comment:
1 sentence description needed - when to use this

Reply:
Done.